_episodes/05-snakemake.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Snakemake"
+teaching: 30
+exercises: 20
+questions:
+- "What are computational workflows?"
+- "How do you program using a build system?"
+- "How do I mix Python code into a workflow?"
+objectives:
+- "Understand the operating logic (or semantics) of Snakemake."
+- "Create a parallel workflow from an existing script."
+keypoints:
+- "Snakemake is ideal for specifying workflows with large granularity."
+- "Dependencies are automatically resolved; Snakemake is 'demand driven' programming."
+- "You may freely intermix shell commands with Python code."
+- "Results are persistent: you can resume computations when needed."
+---
+
+> ## Metalesson!
```

Comment:
I don't think having _metalessons_ goes along very well with the philosophy of The Carpentries. Worst-case scenario, we can tag this as an incomplete episode, and keep working on it (i.e.: this is not a reason for rejecting this pull request).

Reply:
I agree, this is just a push for anyone who doesn't agree to contribute more examples. An instructor can then take their pick. As a counter argument though: anyone teaching this should be experienced enough to be able to setup an example themselves.